adaptor-rm-webapp/src/main/java/com/sample/rm/servlet/Application.java

```diff
@@ -78,6 +83,15 @@
     // End of user code
 
     // Start of user code class_methods
+    @Override
+    public Set<Object> getSingletons() {
+        return Collections.singleton(new AbstractBinder() {
+            @Override
+            protected void configure() {
+                bindFactory(new InmemPagedTrsSingleton()).to(PagedTrs.class);
```

Comment:
Why are referring to the `InmemPagedTrsSingleton` if it shall be injected as `PagedTrs` with help of Jersey?